src/fastoad/openmdao/variables/variable.py

```diff
@@ -348,8 +348,9 @@ def __eq__(self, other):
 
         # Let's also ignore unimportant keys
         for key in METADATA_TO_IGNORE:
-            my_metadata.pop(key, default=None)
-            other_metadata.pop(key, default=None)
+            default_value = None
```

Comment:
Can the default ever be something other than None ? If no, I'd prefer to stay as it was